test-network/network.sh

```diff
@@ -422,6 +437,14 @@ while [[ $# -ge 1 ]] ; do
     CC_INIT_FCN="$2"
     shift
     ;;
+  -cport )
+    CC_SERVER_PORT="$2"
```

Comment:
Since this only applies to chaincode-as-a-service, I'd call it CCAAS_SERVER_PORT to avoid confusion.

Reply:
actually it doesn't exist anymore :-) Not really needed.